graylog2-web-interface/src/pages/SecurityPage.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+
+import * as React from 'react';
+import styled, { css } from 'styled-components';
+
+import { DocumentTitle, IfPermitted, PageHeader } from '../components/common/index';
+import HideOnCloud from '../util/conditional/HideOnCloud';
+import { Col, Row } from '../components/bootstrap/index';
```

Comment:
We can slightly simplify the imports by removing `../` and `/index`.

Reply:
Fixed